### ISSUE TITLE
Batch Normalization: Accelerating Deep Network Training by Reducing Internal Covariate Shift

### ISSUE THREAD

**Issue by jo-kwsm** [jo-kwsm/paper_summary#113]
# INFO
## author
Sergey Ioffe, Christian Szegedy

## affiliation
Google

## conference or year
2015

## link
[arXiv](https://arxiv.org/abs/1502.03167)
[実装](https://paperswithcode.com/paper/batch-normalization-accelerating-deep-network#code)

# 概要

# 提案手法

# 検証

# 新規性

# 議論，展望

# Comment

# date
